composer.json

```diff
@@ -12,14 +12,12 @@
     "require": {
         "php": "^7.2",
         "symfony-cmf/resource-bundle": "^1.0",
-        "jms/serializer-bundle": "^1.0 | ^2.0 | ^3.0",
-        "symfony/translation": "^2.8 || ^3.3 || ^4.0"
```

Comment:
are you sure this is not needed? (totally possible that it was included by mistake, but did you check if any code references things from the Symfony\Components\Translation namespace? i would not trust that the tests in this bundle cover all code)

Reply:
sorry, nevermind. just read #54 :man_facepalming: